src/components/Concept/VisualElement.jsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import Image from '@ndla/ui/lib/Image';
+import loadVisualElement from '../../util/loadVisualElement';
+import { fetchOembed } from '../../api/oembed-proxy/oembedProxyApi';
+import { fetchImage } from '../../api/image/imageApi';
+
+export const getIframeSrcFromHtmlString = html => {
+  const el = document.createElement('html');
+  el.innerHTML = html;
+  const iframe = el.getElementsByTagName('iframe')[0];
+  return iframe.getAttribute('src');
+};
+
+const getFocalPoint = data => {
+  if (data.focalX && data.focalY) {
+    return { x: data.focalX, y: data.focalY };
+  }
+  return undefined;
+};
+
+const getCrop = data => {
+  if (
+    (data.lowerRightX &&
+      data.lowerRightY &&
+      data.upperLeftX &&
+      data.upperLeftY) !== undefined
+  ) {
+    return {
+      startX: data.lowerRightX,
+      startY: data.lowerRightY,
+      endX: data.upperLeftX,
+      endY: data.upperLeftY,
+    };
+  }
+  return undefined;
+};
+
+const VisualElement = visualElement => {
+  const [element, setElement] = useState(null);
+  useEffect(() => {
+    async function getVisualElement() {
+      const parsedElement = await loadVisualElement(
+        visualElement.visualElement,
+      );
+      const data = parsedElement('embed').data();
+      if (data?.resource === 'image') {
+        const image = await fetchImage(data.resourceId);
+        const focalPoint = getFocalPoint(data);
+        const crop = getCrop(data);
+
+        setElement(
+          <Image
+            focalPoint={focalPoint}
+            contentType={image.contentType}
+            crop={crop}
+            alt={data?.alt}
+            src={`${image.imageUrl}`}
+          />,
```

Comment:
Funker med bilde med crop vises ikkje i full bredde. Mulig det må legges på css for bredde.